CSS/style.css

```diff
@@ -0,0 +1,30 @@
+.prodImage {
+  width: 30%;
+  height: 400px;
+  display: inline-block;
+  vertical-align: middle;
+  background-color: #ddd;
```

Comment:
What does background-color on an image look like? Or does this class 'prodImage' reference a container of the image that could accept a background-color?